mep/people/templates/people/member_list.html

```diff
@@ -20,6 +20,23 @@
         <label class="query">
             {% render_field form.query aria-label=form.query.label %}
         </label>
+        <fieldset>
+                <legend>Filter by:</legend>
+                <div class="toggle-filters">
+                    {% render_field form.has_card aria-label=form.has_card.label class+="sr-only sr-only-focusable" %}
+                    <label for="id_has_card">
+                        {# Wrapping attribute in <span> for CSS targeting #}
+                        <span>Card <span class="count">{{ members.get_facets.has_card_b.true }}</span></span>
+                        <img class="filter icon" src="{% static 'img/icons/member_card.svg' %}" alt="card icon">
```

Comment:
I usually give these things a class name according to what the image actually depicts, as you've done for the alt ("card icon"). There should be styles already written for that class since it gets used for the card icons in the results too...might just need to throw on a little extra like `form .card.icon { ....` if it's a slightly different size.